src/components/MobileHome.js

```diff
@@ -1,13 +1,36 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { Tab, Tabs, Paper } from '@material-ui/core';
 import DesktopTabs from './RightPane/RightPaneRoot';
 import Calendar from './Calendar/ScheduleCalendar';
+import RightPaneStore from './RightPane/RightPaneStore';
 
 const MobileHome = () => {
     const [selectedTab, setSelectedTab] = useState(0);
 
     const components = [<Calendar isMobile={true} />, <DesktopTabs style={{ height: 'calc(100% - 50px' }} />];
 
+    const focusOnBuilding = (args) => {
+        // Since MobileHome doesn't have DesktopTabs permamnently loaded,
+        // we need to switch over to it, get a confirmation that it's loaded,
+        // then re-emit 'focusOnBuilding'
+
+        if (selectedTab !== 1) {
+            const reEmitFocus = () => {
```

Comment:
It doesn't cause an infinite loop because, after the first time it runs, it sets `selectedTab` to 1 (SEARCH), which causes the condition to be false. The function does execute twice, but not infinitely